jsonschema-validation.md

```diff
@@ -292,39 +292,46 @@ Structural validation alone may be insufficient to allow an application to
 correctly utilize certain values. The `format` annotation keyword is defined to
 allow schema authors to convey semantic information for a fixed subset of values
 which are accurately described by authoritative resources, be they RFCs or other
-external specifications.
+external specifications. Format values defined externally to this document
+SHOULD also be based on such authoritative resources in order to foster
+interoperability.
 
-The value of this keyword is called a format attribute. It MUST be a string. A
-format attribute can generally only validate a given set of instance types. If
-the type of the instance to validate is not in this set, validation for this
-format attribute and instance SHOULD succeed. All format attributes defined in
-this section apply to strings, but a format attribute can be specified to apply
-to any instance types defined in the data model defined in the [core JSON
-Schema.](#json-schema)[^1]
+The value of this keyword MUST be a string. While this keyword can validate any
+type, each distinct value will generally only validate a given set of instance
```

Comment:
If I'm reading this correctly, it means we let `format` apply to instances other than strings. If I recall correctly, the JSON Schema test suite very explicitly tests that `format` only applies to instances, which might be a change that could confuse various people and implementations.

Reply:
> If I'm reading this correctly, it means we let format apply to instances other than strings.

Yes, `format` can apply to instances of any type.  This isn't a new feature, however, the spec only defines formats that apply to strings.